script/add-grammar

```diff
@@ -84,13 +84,13 @@ if repo_old
   log "Deregistering: #{repo_old}"
```

Comment:
Should we move scripts to the new `tools/` directory or is there a clear distinction between the two (besides the fact that one contains scripting language programs)? If the idea is to always use tools through scripts maybe `tools/` should be a subdirectory (with a new name?) in `script/`...?